### PR TITLE
Croatian translation 20220827

### PR DESCRIPTION
**Pull request by milotype**
Update files with Croatian translation

### CODE REVIEW

**Review by kevinpapst — CHANGES_REQUESTED** [review]
Awesome work, thanks 👍 

There is just one minor issue and then I can merge.

File: _data/menu.yml [hunk 1]
```diff
@@ -23,9 +23,9 @@ fr:
...
+  - informacije
   - demo
-  - download
-  - store
-  - news
-  - docs
+  - preuzimanje
+  - trgovina
+  - novosti
+  - dokumenti
```
[the remark this answers is not in the document]
Comment (milotype) [reply _data/menu.yml]: OK, I just reverted translated identifiers back into english

**Review by kevinpapst — APPROVED** [review]
Thank you so much 👍 looks good, I am going to merge now and then we can do a live chec 😄

**Comment by milotype** [comment]
@kevinpapst Thanks, most of it looks good.

One thing I don't understand is the file **_features/hr/multi-language.md**. I've translated the languages (see https://github.com/kimai/www.kimai.org/pull/144/commits/0d4703f7167c3cfdb3cd5e358fbbd790c5094939), but now they are displayed in english in the featue **Multilingual** at https://www.kimai.org/hr/!?

Can you correct that, or should I make a new PR?

**Comment by kevinpapst** [comment]
I changed that before merging. 

This list is universal across all languages, because I know myself: if I add a language to the list, I forget to add it to all translations ;-)

And one more thought from my side: translating language names isn't really worth the trouble. If you are looking for your own language in a translated page, do you find it more easily to detect it in English or translated in the page language?

Conclusion: let's leave it as is :)

**Pull request merged**